snake_config.yaml

```diff
@@ -0,0 +1,15 @@
+
+idcolumn:
+    gisaid:  gisaid_epi_isl
+    genbank: genbank_accession
+
+
+slack_channel:
+    genbank: ncov-genbank-updates
+    gisaid: ncov-gisaid-updates
```

Comment:
The use of lookup keys like this in config jumps out to me as a sort of anti-pattern. A less brittle approach is to get rid of the lookup keys and instead split one config into multiple, for example:

**genbank.yaml**

```yaml
idcolumn: gisaid_epi_isl
slack_channel: ncov-gisaid-updates
```

**gisaid.yaml**

```yaml
idcolumn: genbank_accession
slack_channel: ncov-genbank-updates
```

Reply:
Could you elaborate on why the 2 files idea is the less brittle approach here ? 

My feeling is that having to maintain multiple  files in parallel, sometimes with redundancies (for all common parameters) is more prone to errors. Provided at the moment the only commonality is the fetch argument, so the difference would not be huge, but still.